CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+### 2.2.0
```

Comment:
Usually we combine beta, rc and stable release notes into one it in the Github release, for Flutter I think it would make sense to combine them in the changelog file as well, since it will be published to pub.dev as is.